### PR TITLE
Update api_callable

### PR DESCRIPTION
**Pull request by jmuk**

### CODE REVIEW

**Comment by jmuk** [comment]
@geigerj, can you take a look?

File: lib/google/gax/api_callable.rb [hunk 1]
```diff
@@ -120,35 +120,35 @@ def next_page_token?
...
                      response_page_token_field, resource_field)
-        @func = a_func
         @request_page_token_field = request_page_token_field
         @page = Page.new(nil, response_page_token_field, resource_field)
       end
 
       # Initiate the streaming with the requests and keywords.
-      # @param page_token [Object]
-      #   The page token for the first page to be streamed, or nil.
+      # @param a_func [Proc]
```
Comment (geigerj) [inline lib/google/gax/api_callable.rb]: s/a_func/api_call

File: lib/google/gax/api_callable.rb [hunk 2]
```diff
@@ -120,35 +120,35 @@ def next_page_token?
...
+      #   A proc to update the response object.
       # @param request [Object]
       #   The initial request object.
+      # @param settings [CallSettings]
+      #   The call settings to enumerate pages.
       # @param kwargs [Hash]
```
Comment (geigerj) [inline lib/google/gax/api_callable.rb]: remove the doc for kwargs

**Comment by geigerj** [comment]
LGTM once PagedEnumerable docs are updated

File: lib/google/gax/api_callable.rb (hunk 1 above)
Comment (jmuk) [inline lib/google/gax/api_callable.rb]: done

File: lib/google/gax/api_callable.rb (hunk 2 above)
Comment (jmuk) [inline lib/google/gax/api_callable.rb]: done

**Pull request merged**